From f3b86b0704cb62b77d80b90b40c1df68289d3558 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Fri, 31 Jan 2014 15:37:24 -0500
Subject: [PATCH] configure: allow subdir objects

autoconf-1.14 will fail to configure lxc with the following warning (and
hence error):

  | automake: warnings are treated as errors
  | src/lxc/Makefile.am:79: warning: source file '../include/openpty.c' is in a subdirectory,
  | src/lxc/Makefile.am:79: but option 'subdir-objects' is disabled

So we tell autoconf that subdir objects are fine .. and the issue is solved.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7dc82cfbcb8d..27ba250df3ac 100644
--- a/configure.ac
+++ b/configure.ac
@@ -6,7 +6,7 @@ AC_INIT([lxc], [0.9.0])
 AC_CONFIG_SRCDIR([configure.ac])
 AC_CONFIG_AUX_DIR([config])
 AC_CONFIG_HEADERS([src/config.h])
-AM_INIT_AUTOMAKE([-Wall -Werror -Wno-portability])
+AM_INIT_AUTOMAKE([-Wall -Werror -Wno-portability subdir-objects])
 AC_CANONICAL_HOST
 AM_PROG_CC_C_O
 AC_GNU_SOURCE
-- 
1.7.10.4

